case_specification.tex

```diff
@@ -0,0 +1,300 @@
+                \draw[->, thick] (nu) -- (enr);
+                \draw[->, thick] (enr) -- (tl);
+                \draw[->, thick] (enr) -- (tl2);
+                \draw[->, thick] (tl) -- (sk);
+                \draw[->, thick] (tl2) -- (mix);
+                \draw[->, thick] (enr) -- (uox);
+                \draw[->, thick] (uox) -- (lwr);
+                \draw[->, thick] (lwr) -- (snf);
+                
+                \draw[->, thick] (lwr) -- (snf);
+                \draw[->, thick] (snf) -- (pool);
+                \draw[->, thick] (pool) -- (cunf);
+                \draw[->, thick] (pool) -- (cunf2);
+                \draw[->, thick] (cunf) -- (sk);
+                \draw[->, thick] (cunf2) -- (rep);
+                
+                \draw[->, thick] (rep) -- (u);
+                \draw[->, thick] (rep) -- (pu);
+                \draw[->, thick] (pu) -- (mix);
+                \draw[->, thick] (mix) -- (mox);
+                \draw[->, thick] (mox) -- (mxr);
+                \draw[->, thick] (mxr) -- (snmox);
+                \draw[->, thick] (snmox) -- (rep);
+                
+                \end{tikzpicture}
+                \end{adjustbox}
+                
+                \caption{Model Fuel Cycle with \gls{MOX} Reprocessing}
+                \label{diag:fc}
+\end{figure}
+\end{frame}
+
+\begin{frame}
+	\frametitle{First Simulation - EU Nuclear Opearations \textasciitilde 2050}
+	Deployment and Reactor data from IAEA PRIS
+	Reprocessing plant and fabrication plant modeled after French La Hague and MELOX site
+	\cite{schneider_spent_2008, hugelmann_melox_1999}.
+		
+\begin{table}[h]
+	\centering
+	\begin{tabularx}{\textwidth}{bb}
+		\hline
+		Specification & Value \\
+		\hline
+		Simulation Time & 1970-2050 \\ 
+		Reprocessing Capacity & 91.6 MTHM of \gls{UNF} per month \cite{schneider_spent_2008} \\
+		Reprocessing Efficiency & 99.8\% \\
+		Reprocessing Streams & Plutonium and Uranium \\
+		\gls{MOX} Fabrication & \small{9\% Reprocessed Pu + 91\% Depleted U} \\
+		\gls{MOX} Fabrication Throughput & 16.25 MTHM of \gls{MOX} per month  \cite{hugelmann_melox_1999} \\
+		\gls{MOX} Fuel Reprocessing Stage &  Used \gls{MOX} gets reprocessed infinitely. \\  
+		Reprocessed Uranium Usage &  None. Stockpile reprocessed U \\
+		\hline
+	\end{tabularx}
+	\caption {Specification for Historical Operation of \gls{EU} Case}
+	\label{tab:sim_eu}
+\end{table}
+	
+\end{frame}
+
+\begin{frame}
+	\frametitle{Second Simulation - French Transition to SFRs \textasciitilde 2160}
+	
+\begin{table}[h]
+	\centering
+	\begin{tabularx}{\textwidth}{bb}
+		\hline
+		Specification & Value \\
+		\hline
+		Simulation Time & 1970-2160 \\
+		\gls{SFR} Available Year & 2040 \\
+		Reprocessing Capacity & $\infty$ \\
+		Reprocessing and Fabrication Begins & 2020 \\
+		Separation Efficiency & 99.8 \% \\
+		Reprocessing Streams & plutonium and uranium \\
+		\small{Used \gls{UOX} and Depleted U Inventory} & 141,659 MTHM {\small (From first simulation)} \\
```

Comment:
Consider including units more systematically in this table. (in their own column, or in brackets after each parameter)